<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>win_snail</class>
 <widget class="QMainWindow" name="win_snail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>789</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string notr="true">win_snail</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/candle_256.png</normaloff>:/images/candle_256.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widgetJog" native="true">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>520</y>
      <width>162</width>
      <height>237</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(128, 255, 128);</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="3">
      <widget class="StyledToolButton" name="cmdYPlus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>44</width>
         <height>44</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Y+</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>resource/chevron-up.png</normaloff>
         <normalon>resource/chevron-up.png</normalon>resource/chevron-up.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="StyledToolButton" name="cmdZMinus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>44</width>
         <height>44</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Z-</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>resource/icon-arrow-down-b.png</normaloff>resource/icon-arrow-down-b.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="StyledToolButton" name="cmdXPlus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>44</width>
         <height>44</height>
        </size>
       </property>
       <property name="toolTip">
        <string>X+</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>resource/chevron-right.png</normaloff>resource/chevron-right.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="StyledToolButton" name="cmdZPlus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>44</width>
         <height>44</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Z+</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>resource/icon-arrow-up-b.png</normaloff>resource/icon-arrow-up-b.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_20">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Step:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="4">
      <widget class="ComboBoxKey" name="cboJogStep">
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::NoInsert</enum>
       </property>
       <item>
        <property name="text">
         <string>Continuously</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0.01</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0.1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>50</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>100</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>200</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="0" colspan="5">
      <widget class="QCheckBox" name="chkKeyboardControl">
       <property name="text">
        <string>Keyboard control</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="StyledToolButton" name="cmdStop">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>36</width>
         <height>36</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Stop</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>resource/brake.png</normaloff>resource/brake.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="StyledToolButton" name="cmdYMinus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>44</width>
         <height>44</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Y-</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>resource/chevron-down.png</normaloff>resource/chevron-down.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="4">
      <widget class="ComboBoxKey" name="cboJogFeed">
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::NoInsert</enum>
       </property>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>50</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>100</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>500</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2000</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="StyledToolButton" name="cmdXMinus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>44</width>
         <height>44</height>
        </size>
       </property>
       <property name="toolTip">
        <string>X-</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>resource/chevron-left.png</normaloff>resource/chevron-left.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>49</y>
      <width>161</width>
      <height>151</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>-21</x>
      <y>0</y>
      <width>611</width>
      <height>761</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="plotterwidget" name="CamWidget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(150, 128, 128);</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(120, 158, 128);</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="buttDebug">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>0</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Debug</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttCon">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>0</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Conn</string>
    </property>
   </widget>
   <widget class="QSlider" name="lightSlider">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>490</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>768</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="mnuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="mnuRecent">
     <property name="title">
      <string>&amp;Recent files</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actRecentClear"/>
    </widget>
    <widget class="QMenu" name="menu_New">
     <property name="title">
      <string>&amp;New</string>
     </property>
     <addaction name="actionProj"/>
     <addaction name="actionFile"/>
    </widget>
    <widget class="QMenu" name="menu_Open">
     <property name="title">
      <string>&amp;Open</string>
     </property>
     <addaction name="actionProj_2"/>
     <addaction name="actionFile_2"/>
    </widget>
    <addaction name="menu_New"/>
    <addaction name="menu_Open"/>
    <addaction name="mnuRecent"/>
    <addaction name="separator"/>
    <addaction name="actFileSave"/>
    <addaction name="actFileSaveAs"/>
    <addaction name="actFileSaveTransformedAs"/>
    <addaction name="separator"/>
    <addaction name="actFileExit"/>
   </widget>
   <widget class="QMenu" name="mnuService">
    <property name="title">
     <string>Camera</string>
    </property>
    <addaction name="actionSelect"/>
   </widget>
   <widget class="QMenu" name="mnuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actAbout"/>
   </widget>
   <widget class="QMenu" name="menuMachine">
    <property name="title">
     <string>Machine</string>
    </property>
    <addaction name="actionMachine_settings"/>
    <addaction name="actionHard_Reset"/>
    <addaction name="actionSet_TLS_Position"/>
    <addaction name="actionDisable_Stepper"/>
   </widget>
   <addaction name="mnuFile"/>
   <addaction name="mnuService"/>
   <addaction name="menuMachine"/>
   <addaction name="mnuHelp"/>
  </widget>
  <action name="actFileExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="text">
    <string>Select</string>
   </property>
  </action>
  <action name="actFileSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actFileSaveAs">
   <property name="text">
    <string>Save &amp;as...</string>
   </property>
  </action>
  <action name="actAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actRecentClear">
   <property name="text">
    <string>&amp;Clear</string>
   </property>
  </action>
  <action name="actFileSaveTransformedAs">
   <property name="text">
    <string>Save &amp;transformed as...</string>
   </property>
  </action>
  <action name="actionMachine_settings">
   <property name="text">
    <string>Machine settings</string>
   </property>
  </action>
  <action name="actionHard_Reset">
   <property name="text">
    <string>Hard Reset</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Perform Hard Reset of machine control&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionSave_TLS_Position">
   <property name="text">
    <string>Set TLS Position</string>
   </property>
  </action>
  <action name="actionSet_TLS_Position">
   <property name="text">
    <string>Set TLS Position</string>
   </property>
  </action>
  <action name="actionDisable_Stepper">
   <property name="text">
    <string>Disable Stepper</string>
   </property>
  </action>
  <action name="actionProj">
   <property name="text">
    <string>Proj</string>
   </property>
  </action>
  <action name="actionFile">
   <property name="text">
    <string>File</string>
   </property>
  </action>
  <action name="actionProj_2">
   <property name="text">
    <string>Proj</string>
   </property>
  </action>
  <action name="actionFile_2">
   <property name="text">
    <string>File</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>StyledToolButton</class>
   <extends>QToolButton</extends>
   <header location="global">styledtoolbutton.h</header>
  </customwidget>
  <customwidget>
   <class>ComboBoxKey</class>
   <extends>QComboBox</extends>
   <header>comboboxkey.h</header>
  </customwidget>
  <customwidget>
   <class>plotterwidget</class>
   <extends>QWidget</extends>
   <header location="global">plotterwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmdXPlus</tabstop>
  <tabstop>cmdYPlus</tabstop>
  <tabstop>cmdYMinus</tabstop>
  <tabstop>cmdZPlus</tabstop>
  <tabstop>cmdZMinus</tabstop>
  <tabstop>chkKeyboardControl</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
